tests/foreman/rhai/test_rhai.py

```diff
@@ -120,14 +120,17 @@ def test_unregister_system_from_rhai(self):
                 self.ak_name,
                 self.org_label,
             )
+
             with Session(self.browser) as session:
                 session.nav.go_to_select_org(self.org_name)
                 Navigator(self.browser).go_to_insights_systems()
                 # Click on the unregister icon 'X' in the table against the
                 # registered system listed.
-                session.nav.click(
-                    locators['insights.unregister_system']
-                )
+                session.nav.wait_for_ajax(timeout=40)
```

Comment:
Yes it does. You can pass the extended timeout direct on the click call. For more info https://github.com/SatelliteQE/robottelo/blob/master/robottelo/ui/base.py#L450